packages/core/src/components/Dialog/_ThirdPartyLink.docs.scss

```diff
@@ -0,0 +1,32 @@
+/*
+Third party external link
+
+The Third party external link pattern is used to notify a user that the action they are taking will take them to a new external website.
+
+Style guide: patterns.external-link
+*/
+
+/*
+`<Dialog />`
+
+@react-component Dialog
+
+@react-example ThirdPartyLink
+
+Style guide: patterns.external-link.react
+*/
+
+/*
+---
+
+Use this pattern when the user is in the middle of a task or application to let them know that the action they are taking will move them away from the current site and or process. This serves as a warning where the user can then decide to remain where they are in the application or process.
+
+### Learn more
+
+- [Making an accessible dialog box](https://www.nczonline.net/blog/2013/02/12/making-an-accessible-dialog-box/)
```

Comment:
are these the same resources on the dialog page? maybe we could link those cms privacy guidance pages?

Reply:
Yep, they are the same. I'm not sure which links you are referring to?

Reply:
Maybe we could replace the resources on dialogs with the links Mel posted on this page:
https://github.com/CMSgov/design-system/issues/499#issuecomment-537659356

Reply:
Ready for another pass